src/Fabulous.AST.Tests/TypeDefinitions/Union.fs

```diff
@@ -32,4 +32,26 @@ type Colors =
     | Blue
     | Yellow
 
+"""
+
+    [<Test>]
+    let ``Produces an union with SingleTextNode`` () =
+        AnonymousModule() {
+            Union(SingleTextNode("Colors", Range.Zero)) {
```

Comment:
Wouldn't a string instead of `SingleTextNode` make more sense?

Reply:
You can do both . This is just a test to verify it . See the test above 😀